mavros_extras/src/plugins/px4flow.cpp

```diff
@@ -149,6 +152,37 @@ class PX4FlowPlugin : public plugin::PluginBase {
 
 		range_pub.publish(range_msg);
 	}
+
+	void send_cb(const mavros_msgs::OpticalFlowRad::ConstPtr msg)
+	{
+		mavlink::common::msg::OPTICAL_FLOW_RAD flow_rad_msg;
+
+		auto int_xy = ftf::transform_frame_baselink_aircraft(
+			Eigen::Vector3d(
+					msg->integrated_x,
+					msg->integrated_y,
+					0.0));
+		auto int_gyro = ftf::transform_frame_aircraft_baselink(

```

Comment:
Why are the transforms for the gyro and the pixel flow different?

Reply:
Copy-paste issue again :(